eng/scripts/Language-Settings.ps1

```diff
@@ -640,3 +640,49 @@ function Get-python-EmitterName() {
 function Get-python-EmitterAdditionalOptions([string]$projectDirectory) {
   return "--option @azure-tools/typespec-python.emitter-output-dir=$projectDirectory/"
 }
+
+function Get-python-DirectoriesForGeneration () {
+  Get-ChildItem "$RepoRoot/sdk" -Directory
```

Comment:
Just style wise I suggest an explicit return here to avoid later gotchas, especially for people that may not be familiar with the nuances of powershell returns.

Reply:
updated